README.md

```diff
@@ -46,57 +34,274 @@ npm start
 
 This will start hops in development mode. Visit [http://localhost:8080](http://localhost:8080) to see your app in the browser and make some changes to the code in your editor to see it live-reloading.
 
-You can find a complete walk through in our [Getting Started Guide](https://github.com/xing/hops/wiki/getting-started).
+### Templates
+
+Hops provides a few templates as starting points, which can be installed using the `--template` option of the [Hops CLI `init` command](https://github.com/xing/hops/tree/next/packages/cli).
+
+#### [`hops-template-react`](https://github.com/xing/hops/tree/next/packages/template-react) (default)
+
+This is the default template, it consists of a simple React starting point and it is being used when you execute:
+
+```bash
+$ hops init my-awesome-project
+```
+
+#### [`hops-template-redux`](https://github.com/xing/hops/tree/next/packages/template-redux)
+
+In case you want to build a Redux project, you can use this template as a starting point.
+
+```bash
+$ hops init my-awesome-project --template hops-template-redux
+```
+
+#### [`hops-template-graphql`](https://github.com/xing/hops/tree/next/packages/template-graphql)
+
+In case you want to build a GraphQL project, you can use this template as a starting point.
+
+```bash
+$ hops init my-awesome-project --template hops-template-graphql
+```
+
+#### [`hops-template-minimal`](https://github.com/xing/hops/tree/next/packages/template-minimal)
+
+This template is not seriously meant as a starting point for your project, but rather a reference that demonstrates using Hops without React and might be useful to someone trying to create Vue.js bindings for Hops or something similar.
+
+## Presets
+
+Almost everything in Hops is a preset that just needs to be installed / configured in order to start using it.
```

Comment:
I'd go through all the .md files and remove all occurences of `just` / `easy` and the like. 

It might appear trivial, but it isn't for everyone and we shouldn't make people feel bad, by having the text suggests that it is easy as pie.